ui/src/app/pages/data/concept/concept-add-modal.tsx

```diff
@@ -148,11 +193,19 @@ export const ConceptAddModal = withCurrentWorkspace()
     }
   }
 
+  get Title() {
+    const {selectedDomain, selectedSurvey, surveyName} = this.props;
+    const {selectedConceptsInDomain} = this.state;
+
+    return !this.isSurvey() ? 'Add ' + selectedConceptsInDomain.length + ' Concepts to ' +
```

Comment:
There's a lot of shared language here. The only different bits seem to be `Concepts` vs `Survey` and the name vs the number. Can we just parameterize that and keep the shared language?